### PR TITLE
feat(CC): add two CC resource 

### PR DESCRIPTION
**Pull request by hmbig2**
**What this PR does / why we need it**:

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note

```

## PR Checklist

* [X] Tests added/passed.
* [X] Documentation updated.
* [X] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST='./huaweicloud/services/acceptance/cc' TESTARGS='-run=TestAccCloud'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/cc -v -run=TestAccCloud -timeout 360m -parallel 4
=== RUN   TestAccCloudConnection_basic
=== PAUSE TestAccCloudConnection_basic
=== CONT  TestAccCloudConnection_basic
--- PASS: TestAccCloudConnection_basic (20.88s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/cc        20.932s
```

```
make testacc TEST='./huaweicloud/services/acceptance/cc' TESTARGS='-run=TestAccNetworkInstance_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/cc -v -run=TestAccNetworkInstance_basic -timeout 360m -parallel 4
=== RUN   TestAccNetworkInstance_basic
=== PAUSE TestAccNetworkInstance_basic
=== CONT  TestAccNetworkInstance_basic
--- PASS: TestAccNetworkInstance_basic (57.30s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/cc        57.350s
```

### CODE REVIEW

File: docs/resources/cc_connection.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+---
+subcategory: Cloud Connect (CC)
+---
+
+# huaweicloud_cc_connection
+
+Manages a Cloud Connection resource within HuaweiCloud.  
+Cloud Connect (CC) is a service that enables you to quickly build ultra-fast, high-quality, and stable networks
+between VPCs across regions and between VPCs and on-premises data centers.
```
Comment (Jason-Zhang9309) [inline docs/resources/cc_connection.md:10]: duplicate `between VPCs`?

File: docs/resources/cc_connection.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+* `name` - (Required, String) The cloud connection name.  
+  The name can contain 1 to 64 characters, only letters, Chinese characters, digits, hyphens (-),
+  underscores (_) and dots (.).
+
+* `description` - (Optional, String) The Description about the cloud connection.  
+  The description can contain a maximum of 255 characters.
+
+* `enterprise_project_id` - (Optional, String, ForceNew) The enterprise project id of the cloud connection.  
+  Value 0 indicates the default enterprise project.
+
```
Comment (Jason-Zhang9309) [inline docs/resources/cc_connection.md:35]: extral blank line

File: docs/resources/cc_connection.md (hunk 1 above)
Comment (hmbig2) [reply docs/resources/cc_connection.md:10]: This sentence comes from CC's official documentation.  https://support.huaweicloud.com/intl/en-us/cc/
so keep it

File: huaweicloud/services/cc/resource_huaweicloud_cc_connection.go [hunk 3]
```diff
@@ -0,0 +1,306 @@
...
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `The Domain ID.`,
+			},
+			"status": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `The status of the cloud connection.`,
+			},
+			"admin_state_up": {
```
Comment (ShiChangkuo) [inline huaweicloud/services/cc/resource_huaweicloud_cc_connection.go]: admin_state_up should be deleted in schema and docs

File: docs/resources/cc_network_instance.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+  underscores (_) and dots (.).
+
+* `description` - (Optional, String) The description about the network instance.  
+  The description can contain a maximum of 255 characters.
+
+* `instance_domain_id` - (Optional, String, ForceNew) Account ID of the VPC or virtual gateway.
+
+  Changing this parameter will create a new resource.
+
+* `cidrs` - (Optional, List) List of routes advertised by the VPC or virtual gateway.
```
Comment (ShiChangkuo) [inline docs/resources/cc_network_instance.md]: it's better to put parameters of instance together.

File: docs/resources/cc_connection.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
+---
+subcategory: Cloud Connect (CC)
+---
+
+# huaweicloud_cc_connection
+
+Manages a Cloud Connection resource within HuaweiCloud.  
+Cloud Connect (CC) is a service that enables you to quickly build ultra-fast, high-quality, and stable networks
```
Comment (ShiChangkuo) [inline docs/resources/cc_connection.md:9]: please add a new line

**Pull request merged**